app/seed-data/formula_values/data_processed/EFDB_2006_IPCC_guidelines/FormulaInputs.csv

```diff
@@ -1,4 +1,4 @@
-gas,parameter_code,parameter_name,methodology,gpc_refno,year,formula_input_value,formula_input_units,formula_name,metadata,Region,actor_id,datasource,RNK,methodology_id,formulainput_id
+gas,parameter_code,parameter_name,methodology_name,gpc_refno,year,formula_input_value,formula_input_units,formula_name,metadata,region,actor_id,datasource,rnk,methodology_id,formulainput_id
```

Comment:
These also need to be changed in the python script. We can do that in this PR or ask the data team to do it